src/components/experiment-tracking/run-dataset/run-dataset.test.js

```diff
@@ -3,48 +3,56 @@ import RunDataset from '.';
 import { runs, trackingData } from '../../experiment-wrapper/mock-data';
 import { shallow, mount } from 'enzyme';
 
-const booleanTrackingData = [
-  {
-    datasetName: 'Data Analysis',
-    data: {
-      classWeight: [{ runId: 'My Favorite Sprint', value: false }],
+const booleanTrackingData = {
+  json: [
```

Comment:
Should this be `JSONData` just to match the query?

Also do we want test cases that include metrics?